### ISSUE TITLE
Getting :  IAS ERROR: Could not build any suitable CRTC's

### ISSUE THREAD

**Issue by weiseng-yeap** [clearlinux/distribution#1809]
Hi,

I using the Clear Version :32510, running on Intel NUC7i7BNH platform.

When trigger the command to enable ias get error.

Command : $systemctl start ias

Error:
               EGL Wayland extension: yes
[15:57:32.209] Found 0 matching sets of display components to build
[15:57:32.209] IAS ERROR: Could not build any suitable CRTC's
[15:57:32.209] failed to create crtcs for /sys/devices/pci0000:00/0000:00:02.0/drm/card0
[15:57:32.212] fatal: failed to create compositor backend

Please help provide the tips to solve this issue.

**Comment by gvancuts** [comment]
Have you made any changes to the default config? (e.g. in `/usr/share/xdg/weston/ias.conf`) this is where I would start and also check if there are further information in `/tmp/ias.log`

For information about your CRTC, take a look inside `/sys/kernel/debug/dri/0/i915_display_info`

**Comment by weiseng-yeap** [comment]
$/sys/kernel/debug/dri/0/i915_display_info | grep " connected" | cut -d' ' -f4 | head -1

Response:
DP-1

So I change the /usr/share/xdg/weston/ias.conf
to DP-1

I not get error, resolve.

But last time I using Clear 28100 is dos based when trigger the $systemctl start ias can able to see the IAS Weston screen show.

For now I using the Clear with Desktop, when I trigger the $systemctl start ias I cannot get the ias Weston screen. How to enable it?

**Comment by gvancuts** [comment]
I don't think you can start ias if you're already running a desktop environment, can you try to disable it, reboot and try again to restart the `ias.service`? (IIRC, removing the `desktop-autostart` bundle will prevent the desktop from starting. Masking the `gdm` service may also do the same)

**Comment by weiseng-yeap** [comment]
Its possible to get the bare metal Clear version? 

Or can you provide me the guide for this : RC, removing the desktop-autostart bundle will prevent the desktop from starting. Masking the gdm service may also do the same) ?

Appreciate!

**Comment by gvancuts** [comment]
> Its possible to get the bare metal Clear version?

Do you mean remove entirely the desktop environment?

> 
> Or can you provide me the guide for this : RC, removing the desktop-autostart bundle will prevent the desktop from starting. Masking the gdm service may also do the same) ?

`sudo swupd bundle-remove dektop-autostart`
`sudo systemctl mask gdm`

And then reboot

**Comment by weiseng-yeap** [comment]
Thanks!
Simple question:
sudo swupd bundle-remove dektop-autostart
sudo systemctl mask gdm

After above command will change to using terminal console only?

Appreciate!

**Comment by gvancuts** [comment]
Yes (you only need to use one of them)

**Comment by weiseng-yeap** [comment]
I will try next Monday, due to Friday night here. Have good weekend to you!! Very great support!!
